config/initializers/devise.rb

```diff
@@ -234,10 +234,14 @@
   # up on your models and hooks.
   # config.omniauth :github, 'APP_ID', 'APP_SECRET', scope: 'user,public_repo'
 
-  config.omniauth :twitter,
-    Rails.application.secrets.twitter_api_key,
-    Rails.application.secrets.twitter_api_secret
-
+  config.omniauth :twitter, Rails.application.secrets.twitter_api_key, Rails.application.secrets.twitter_api_secret
+
+  if Rails.application.secrets.github_app_id.present?
+    config.omniauth :github,
+      Rails.application.secrets.github_app_id,
+      Rails.application.secrets.github_app_secret,
+      scope: 'user'
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.